tests/framework/caching/OciCacheTest.php

```
@@ -0,0 +1,74 @@
+
+    /**
+     * @param bool $reset whether to clean up the test database
+     *
+     * @return Connection
+     */
+    public function getConnection($reset = true)
+    {
+        if ($this->_connection === null) {
+            $databases = self::getParam('databases');
+            $params = $databases['oci'];
+            $db = new Connection();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)